src/components/views/messages/MFileBody.tsx

```diff
@@ -274,20 +230,6 @@ export default class MFileBody extends React.Component<IProps, IState> {
                             { /* eslint-disable-next-line */ }
                             <a ref={this.dummyLink} />
                         </div>
-                        { /*
-                            TODO: Move iframe (and dummy link) into FileDownloader.
-                            We currently have it set up this way because of styles applied to the iframe
-                            itself which cannot be easily handled/overridden by the FileDownloader. In
-                            future, the download link may disappear entirely at which point it could also
-                            be suitable to just remove this bit of code.
-                         */ }
-                        <iframe
-                            aria-hidden
-                            title={presentableTextForFile(this.content, _t("Attachment"), true, true)}
-                            src={url}
-                            onLoad={() => this.downloadFile(this.fileName, this.linkText)}
-                            ref={this.iframe}
-                            sandbox="allow-scripts allow-downloads allow-downloads-without-user-activation" />
```

Comment:
Pretty sure this iframe is still needed, this.iframe is passed into this.fileDownloader's c'tor

Reply:
As discussed in chat, FileDownloader creates an equivalent iframe when none is passed to the constructer. Removed it from the ctor here, and the ref.